nav2_costmap_2d/include/nav2_costmap_2d/exceptions.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2017, Locus Robotics
```

Comment:
Add:
Copyright (c) 2019, Intel Corporation

Reply:
This looks like mostly copied code. I wouldn't change the copyright.